package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@launchpadlab/lp-components",
-  "version": "5.4.2",
+  "version": "5.4.4",
```

Comment:
Always base the version bump off of your base branch, not other PRs. This could very likely get merged before #503